packages/driver/src/dom/elements.ts

```diff
@@ -889,77 +909,49 @@ const getActiveElByDocument = (doc: Document): HTMLElement | null => {
 }
 
 const getFirstParentWithTagName = ($el, tagName) => {
-  // return null if undefined or we're at body/html/document
-  // cuz that means nothing has fixed position
-  if (isUndefinedOrHTMLBodyDoc($el) || !tagName) {
-    return null
-  }
-
-  // if we are the matching element return ourselves
-  if (getTagName($el[0]) === tagName) {
-    return $el
```

Comment:
`$el.get(0)` versus `$el[0]`... are these equivalent?

Reply:
yup afaik jquery's get() just indexes into its self behind the scenes. we should be consistent though

Reply:
saying that though this file is littered with both forms so there isn't really a "consistent" one yet 🙈